app.py

```diff
@@ -808,6 +833,6 @@ def update_yaml():
 if __name__ == '__main__':
     # Threaded option to enable multiple instances for multiple user access support
     if version() == 'DEV':
-        app.run(threaded=True, port=config ['port'], host="0.0.0.0")
+        app.run(threaded=True, debug=True, port=config ['port'], host="0.0.0.0")
```

Comment:
I'm also curious as to why we're not currently running in debug mode locally. I added this, since talisman won't force HTTPS when in debug mode, so this should allow us to use talisman and still run fine locally.

Reply:
Oh that's a good call. I generally start the server with `FLASK_ENV=development`, but you're right.

In fact, I think this whole block can be simplified a lot, because it will only ever be executed for local setups anyway. For production setups, we'll go through the `Procfile` which doesn't execute this block. So yeah, always defaulting to `debug=True` makes sense.

Not sure why we do the check of `version()` -- @fpereiro, do you think this will ever take the alternative branch/

Reply:
HI @lpmi-13 & @rix0rrr -- this block would be necessary unless `__name__` was not equivalent to `'__main__'` - if we remove it, wouldn't it be executed in Heroku as well?

:+1: to run debug=True locally.

Reply:
I meant the `if version() == 'DEV'` condition:

https://github.com/Felienne/hedy/blob/9e7a2e7f3fcaf360f774d429e2f42b9bec18bb80/app.py#L877

In Heroku, the command that is executed is:

```
$ gunicorn app:app
```

(Which you can see in the `Procfile`).

`if __name__ == '__main__'` is only true if the script in question is the one that is started directly, which in the case of `gunicorn` is not true.

So I think we could simplify this whole block to:

```python
if __name__ == '__main__':
    app.run(threaded=True, debug=True, port=config['port'], host="0.0.0.0")
```

Reply:
Ok, then we can definitely simplify it :+1:

Reply:
will update as suggested!